packages/gutenframe/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "@automattic/gutenframe",
+	"version": "1.0.0-alpha.0",
+	"description": "Utilities for the block editor iframe connections",
+	"main": "dist/server.js",
+	"sideEffects": false,
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/Automattic/wp-calypso.git",
+		"directory": "apps/gutenframe"
+	},
+	"private": true,
+	"author": "Automattic, Inc.",
+	"license": "GPL-2.0-or-later",
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"scripts": {
+		"clean": "npx rimraf dist",
+		"prebuild": "npm run clean",
+		"prepare": "webpack"
```

Comment:
@ockham @sirreal is it possible to use external WP stuff by relying directly to calypso-build package and drop webpack/babel configs here?

Reply:
I suspect we can drop the webpack config and use it just like o2-blocks:

https://github.com/Automattic/wp-calypso/blob/6fb40ceab56241c4c2195cc71c2a9ef9cbb0d13e/apps/o2-blocks/package.json#L22

Reply:
Keep an eye on https://github.com/Automattic/wp-calypso/pull/32042 as well, `env.WP='true'` will need to be passed to webpack

Reply:
I removed the webpack config file when I moved the package to the `apps` dir (521131d).

But I was not able to remove the babel config. If I remove it, I get this error:

<img width="1030" alt="Screen Shot 2019-04-10 at 08 48 47" src="https://user-images.githubusercontent.com/1233880/55842218-7a202280-5b6d-11e9-8d17-ff6374acb32a.png">

Reply:
@ockham ☝️